Framework/PythonInterface/plugins/algorithms/LoadWANDSCD.py

```diff
@@ -48,46 +117,104 @@ def validateInputs(self):
         return issues
 
     def PyExec(self):
-        runs = self.getProperty("Filename").value
-
-        if not runs:
+        # Process input data
+        # get the list of input filenames
+        runs = self.get_intput_filenames()
+        # load and group
+        data = self.load_and_group(runs)
+
+        # Process the vanadium normalization data
+        if self.getProperty("NormalizedBy").value.lower() == 'none':
+            # no normalization
+            print("Skip Va normalization as requested")
```

Comment:
You should change all the `print`'s to use the Mantid logging, _e.g._ `self.getLogger().information("hello")`

Reply:
got it, replacing it now